saas/static/js/djaodjin-saas-vue.js

```diff
@@ -1870,18 +2008,18 @@ new Vue({
                 for(var key in data){
                     form.append(key, data[key]);
                 }
-                vm.req({
+                $.ajax({
```

Comment:
Please add a comment here why we are not using one of the `reqVerb` functions but `$.ajax` directly (ex: "We are using `$.ajax` directly here because this requires to pass the parameters as url encoded (multipart) instead of the normal data type `application/json`.")